//package com.sincetimes.redwar.game.comnunicate.protocol;


message RseItemDamaged
{
    optional int32 ret = 1;
}

